Meta/lint-commit.sh

```diff
@@ -24,6 +24,8 @@ while read -r line; do
 
   # ignore comment lines
   [[ "$line" =~ ^#.* ]] && continue
```

Comment:
Typo in the comment, shouldn't it be `fixup! `? Here and in the action edit.

Reply:
Thanks! I can't believe ~~it's not butter~~ that I overlooked it twice.